simulation-runner/config/scenarios/ApplesToApples.json

```diff
@@ -0,0 +1,12 @@
+{
+    "run_description": "'role::by_size_large' role from perf control repo, 750 agents, 8 iterations",
```

Comment:
should the file be named with 750/8 somehow?  or do we think we might not have other scenarios saved?  YAGNI, etc

Reply:
My thought was that scenarios should be something generic that can be run over and over again, like whatever configuration we use for apples to apples.  Down the road, we may end up changing what we want to use for apples to apples, and I'd rather not have to deal with also changing the file name?

Reply:
yeah, but i'm also fine if we ticket to fix this up later.  we have other tickets to tackle in this sprint 🤷‍♂️

Reply:
that actually makes good sense.  Lets just be sure to get it ticketed.

Reply:
@RandellP Good point about the 600/900. I'll change it to 600. 
I still disagree about the filename. The way it is currently, its easy to change what we want an ApplesToApples run to actually do without much trouble, just update the file contents. If we match the filename with the number of instances and repetitions, then any time we want to update the scenario, we have to change the file contents, file name and references to the file, and it also makes it harder to figure out which scenario we use for the apples to apples run without looking into the code to figure out which scenario is called instead of just finding the scenario named ApplesToApples.

Reply:
We can ticket the idea of changing the name be a discussion for later.  I understand your points, and they are good.  I have some long term thoughts that may end up not mattering, or might be a very good reason... :)  But we can hash that out when we groom the ticket and see where we end up.